vms/platformvm/txs/executor/state_changes.go

```diff
@@ -58,30 +57,6 @@ func VerifyNewChainTime(
 	return nil
 }
 
-func NextBlockTime(state state.Chain, clk *mockable.Clock) (time.Time, bool, error) {
```

Comment:
`NextBlockTime` is the function we need to reuse e.g. in txBuilder package, when dynamic fees are activated.
We move it to state package to avoid circular dependencies and unlock compilation